readme.md

```diff
@@ -106,6 +106,7 @@ Thanks for contributing! 🦋🙌
 - [](# "link-to-github-io") [Adds a link to visit the user’s github.io website from its repo.](https://user-images.githubusercontent.com/31387795/94045261-dbcd5e80-fdec-11ea-83fa-30bb673cc26e.jpg)
 - [](# "next-scheduled-github-action") [Shows the next scheduled time of relevant GitHub Actions in the workflows sidebar.](https://user-images.githubusercontent.com/46634000/94690232-2476a180-0330-11eb-99d7-e174bb762cea.png)
 - [](# "quick-repo-deletion") [Lets you delete your repos in a click, if they have no stars, issues, or PRs.](https://user-images.githubusercontent.com/1402241/99716945-54a80a00-2a6e-11eb-9107-f3517a6ab1bc.gif)
+- [](# "useful-forks") [Adds a button to find relevant forks quickly.](https://user-images.githubusercontent.com/38117856/107256825-fe60bc80-6a07-11eb-93d5-7c3e8e20e922.png)
```

Comment:
```suggestion
- [](# "useful-forks") [Helps you find the most active forks.](https://user-images.githubusercontent.com/38117856/107256825-fe60bc80-6a07-11eb-93d5-7c3e8e20e922.png), via https://useful-forks.github.io.
```

Also I would include more of the UI in the screenshot so what's being pictured is clearer:

<img width="1039" alt="Screen Shot 1" src="https://user-images.githubusercontent.com/1402241/107274269-b8aaf080-6a15-11eb-8388-f61fa600c148.png">

An arrow may be necessary to point to the new button. Any style works, you can see other screenshots for reference.

If possible, zoom the page first so that the screenshot is in higher resolution.